README.md

```diff
@@ -42,9 +42,7 @@ nexusPublishing {
 ```gradle
 nexusPublishing {
     repositories {
-        sonatype {
-            nexusHost = s01
-        }
+        sonatype(s01)
```

Comment:
Doesn't it require any import? That's a reason, I am reluctant in using enum directly. I remember situations in Groovy build.gradle where Idea shows red error on an import (even thought, it was correctly resolved by Gradle itself). Has it improved over time?

Reply:
`s01` is a member of the `NexusRepositoryContainer` interface